src/session/src/context.rs

```diff
@@ -518,7 +518,7 @@ pub enum Channel {
 
     Mysql = 1,
     Postgres = 2,
-    Http = 3,
+    Httpsql = 3,
```

Comment:
```suggestion
    HttpSql = 3,
```

Reply:
Should we change all of them to this style, like PromQL ElasticSearch